inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -378,46 +376,42 @@ discretized Gamma.
 
 ## Results
 
-```{r, eval = params$use_epiparameter}
-si_epiparameter <- epiparameter::epidist(
-  params$epiparameter_pathogen,
-  "serial_interval"
+```{r, eval = params$use_epiparameter_database}
+si_epidist <- epiparameter::epidist_db(
+  disease =  params$disease_name,
+  epi_dist =  "serial_interval",
+  single_epidist = TRUE,
+  subset = is_parameterised
 )
-si_params <- si_epiparameter$param
-si_dist <- si_epiparameter$dist
-si_mean <- si_params[[1]]
-si_sd <- si_params[[2]]
+
+si_params <- get_parameters(si_epidist)
+si_dist <- family(si_epidist)
+si_mean <- get_parameters(si_epidist)[1]
+si_sd <- get_parameters(si_epidist)[2]
 ```
 
-```{r, eval = !params$use_epiparameter}
+```{r, eval = !params$use_epiparameter_database}
 si_mean <- params$si_mean
 si_sd <- params$si_sd
 si_dist <- params$si_dist
 si_cv <- si_sd / si_mean
```

Comment:
It seems that `si_cv` is being defined but never used. If never used it can be removed.

Reply:
@Bisaloo as the person who originally wrote the report and created the object, what was the purpose of this object? can I remove it from the script?

Reply:
Looks like it was previously used in the following line but it's no longer the case so I would indeed remove this object now.

Reply:
I've removed it here https://github.com/epiverse-trace/episoap/pull/100/commits/e5a9108fa5739e1d6b7e544cffe3ef3440951152